scripts/build_docs.py

```diff
@@ -31,10 +34,13 @@ def render_file(relative_src_path: str, src_file: str, to_file: str, modifier="+
     else:
         namespace = f"allennlp.{relative_src_namespace}.{src_base}{modifier}"
 
-    args = ["mathy_pydoc", namespace]
-    call_result = check_output(args, env=os.environ).decode("utf-8")
+    sys.argv = ["mathy_pydoc", namespace]
+    output = io.StringIO()
+    with redirect_stdout(output):
+        main()
+
     with open(to_file, "w") as f:
-        f.write(call_result)
+        f.write(output.getvalue())
```

Comment:
If it just gets written to a file, why not do `with redirect_stdout(f):`?

Reply:
That seems to work ;-)

Reply:
We're down to 5s now...

Reply:
Pretty soon this script will finish before it started.